release/rllib_tests/multi_node_checkpointing_compute_config_gce.yaml

```diff
@@ -0,0 +1,22 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
```

Comment:
this block is also for aws as well

Reply:
Totally, I think the config for gce looks something like this


```
cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
region: us-west1
allowed_azs: 
    - us-west1-c
```

Reply:
azs us west1 b or c?

Reply:
b was for GPU workloads?

Reply:
yes, b sounds good!